src/main/resources/application-dev.yml

```diff
@@ -27,7 +27,7 @@ hmpps-auth:
 nomis:
   endpoint:
     url: http://localhost:8082
-    timeout: 1s
+    timeout: 3m
```

Comment:
this will increase the timeout on all requests to nomis user roles - probably best to have a separate webclient config for the download request?